azure-pipelines.yml

```
@@ -0,0 +1,18 @@
+jobs:
+- job: macOS
+  pool:
+    vmImage: xcode9-macos10.13
+  steps:
+    - bash: |
+        export HOMEBREW_COLOR=1
+        export HOMEBREW_DEVELOPER=1
+        export HOMEBREW_NO_AUTO_UPDATE=1
+        brew update
+        # Mirror the repo as a tap.
+        HOMEBREW_TAP_DIR='/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask'
+        mkdir -p "${HOMEBREW_TAP_DIR}"
+        rsync -az --delete "${HOMEBREW_TAP_DIR}"
```

```suggestion
        rsync --archive --compress --delete "${HOMEBREW_TAP_DIR}"
```

Again, we should use long flags whenever they exist.

---

my fault sorry

---

Its still bringing the same error :(

---

> Its still bringing the same error :(

Note the changes I’m suggesting are not to fix the errors (that’s the area @reitermarkus), but to tidy everything up consistently.

---

Oops. Sure. Thanks for the help. 👍